cinema/models.py

```diff
@@ -84,30 +84,35 @@ class Ticket(models.Model):
     row = models.IntegerField()
     seat = models.IntegerField()
 
-    def clean(self):
+    @staticmethod
+    def validate_seat(row, seat, movie_session, error_to_raise):
         for ticket_attr_value, ticket_attr_name, cinema_hall_attr_name in [
-            (self.row, "row", "rows"),
-            (self.seat, "seat", "seats_in_row"),
+            (row, "row", "rows"),
+            (seat, "seat", "seats_in_row"),
         ]:
-            count_attrs = getattr(
-                self.movie_session.cinema_hall, cinema_hall_attr_name
-            )
+            count_attrs = getattr(movie_session.cinema_hall,
+                                  cinema_hall_attr_name)
             if not (1 <= ticket_attr_value <= count_attrs):
-                raise ValidationError(
-                    {
-                        ticket_attr_name: f"{ticket_attr_name} "
-                        f"number must be in available range: "
-                        f"(1, {cinema_hall_attr_name}): "
-                        f"(1, {count_attrs})"
-                    }
-                )
+                raise error_to_raise({
+                    ticket_attr_name: (
+                        f"{ticket_attr_name.capitalize()} "
+                        f"number must be within the range "
+                        f"(1, {count_attrs})."
+                    )
+                })
+
+    def clean(self):
+        Ticket.validate_seat(self.row,
+                             self.seat,
+                             self.movie_session,
+                             ValueError)
```

Comment:
In the `validate_seat` method, the `error_to_raise` parameter is set to `ValueError`. However, it might be more appropriate to use `ValidationError` from `django.core.exceptions` for validation-related errors, as this is more consistent with Django's validation framework.